registration-processor-default.properties

```diff
@@ -642,7 +643,7 @@ mosip.regproc.demo.dedupe.message.expiry-time-limit=${mosip.regproc.common.stage
 mosip.regproc.demo.dedupe.server.port=8091
 mosip.regproc.demo.dedupe.eventbus.port=5717
 mosip.regproc.demo.dedupe.server.servlet.path=/registrationprocessor/v1/demodedupe
-mosip.regproc.demo.dedupe.trim-whitespaces.simpleType-value=false
+mosip.regproc.demo.dedupe.trim-whitespaces.simpleType-value:false
```

Comment:
We should use mosip.regproc.demo.dedupe.trim-whitespaces.simpleType-value=false

Reply:
Done